src/app/helptext/vm/vm-wizard/vm-wizard.ts

```diff
@@ -77,7 +77,8 @@ volsize_validation: [Validators.required, Validators.min(1)],
 
 pool_detach_warning_paraText: T("Select zvol"),
 
-datastore_tooltip: T('Choose a pool or dataset for the new zvol.'),
+datastore_tooltip: T('Select where to create the new zvol.'),
+datastore_placeholder: T('Zvol Location'),
```

Comment:
The file browser allows selecting locations that are not datasets, then fails because it requires a dataset.  The file browser here should only allow choosing datasets.
```suggestion
datastore_placeholder: T('Zvol Dataset Location'),
```